### PR TITLE
Read carbon capture data from new DEA industry sheets

### PR DESCRIPTION
**Pull request by nworbmot**
Read in for direct air capture and post combustion capture on a small biomass CHP and a cement kiln.

### CODE REVIEW

**Review by lisazeyen — APPROVED** [review]
We can merge this branch. In the long-term I would like to

(1) remove the extra function add_carbon_capture(data) and rather include processing the data in the previous steps   
(2) add as well the costs for gas CHPs CCS/ carbon capture from p.86 [DEA version 0009](https://ens.dk/sites/ens.dk/files/Statistik/technology_data_catalogue_for_el_and_dh_-_0009.pdf)
(3) than remove the function add_costs_ccs(costs) which currently adds costs for CCS from DIW

File: scripts/compile_cost_assumptions.py [hunk 1]
```diff
@@ -82,6 +82,9 @@
                'biogas upgrading': '82 Biogas, upgrading',
                'battery': '180 Lithium Ion Battery',
                'electrolysis': '88 Alkaline Electrolyser',
+               'direct air capture' : '403.a Direct air capture',
+               'biomass CHP capture' : '401.a Post comb - small CHP',
+               'cement capture' : '401.c Post comb - Cement kiln',
```
[the remark this answers is not in the document]
Comment (martavp) [reply scripts/compile_cost_assumptions.py:87]: Yes, it does, in section '07 CO2 Capture and Storage'

**Comment by martavp** [comment]
I agree. We can merge this branch, but @nworbmot maybe want to address @lisazeyen before merging

**Comment by nworbmot** [comment]
Hi both, yes, I agree with @lisazeyen's points. I'm just tidying up the scripts a bit now. I have removed the redundant function "add_costs_ccs", i.e. point (3). That doesn't address points (1) and (2) but it's a start. Will push soon, then merge, then do a release to coincide with PyPSA-Eur-Sec v0.4.0.

**Pull request merged**